src/it/java/org/jboss/tools/intellij/quarkus/utils/BuildUtils.java

```diff
@@ -34,6 +34,7 @@ public static void buildTheProject(RemoteRobot remoteRobot, ToolToBuildTheProjec
                 case MAVEN:
                     step("Open the Maven tab and build the project", () -> {
                         final ToolWindowsPaneFixture toolWindowsPaneFixture = remoteRobot.find(ToolWindowsPaneFixture.class);
+                        GlobalUtils.takeScreenshot();
```

Comment:
Is this really required ? I thought screenshots were for errors

Reply:
Sorry, I forgot that there while testing, it should of course be removed -> deleted.